app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,45 @@
+class PostsController < ApplicationController
+  def index
+    @user = User.includes(:posts).find(params[:user_id])
+    @posts = @user.posts
```

Comment:
- [ ] Dear Team, I notice you have used the `.icludees(:posts)` to fix the N+! problem, this is great. But please be reminded the requirement states `N+1 problem is solved when fetching all posts and their comments for a user`, which means we need to resolve the `N+1` when fetching the comments for the posts too. Please apply the same technique to fetch the `comments` for the posts.